### ISSUE TITLE
One click download

### ISSUE THREAD

**Issue by MartijnCuppens** [Johann-S/bs-customizer#24]
I don't think the modal has an added value, shouldn't we just add a loading state to the build-button and rename it to download?

**Comment by Johann-S** [comment]
The value I see in the modal, it's it disable inputs thanks to the `backdrop` so if we remove the modal we should disable all the checkbox during the build 🤔

**Comment by MartijnCuppens** [comment]
We could add an overlay over the form? or add `pointer-events: none` to to form?

**Comment by Johann-S** [comment]
Plus I like the modal because we can display some text which explain what we are doing (that's an improvement I have in mind since the beginning), for example : Downloading files --> Building files --> Minifying --> Ready

I think it's important to give a lot of information about what we're doing to the user, so if you have any idea how to do that with or without a modal, do not hesitate 👍

**Comment by MartijnCuppens** [comment]
We can display this in or below the button? The thing that _"bothers"_ me is that I need to click, wait and click again to download the file.

**Comment by MartijnCuppens** [comment]
The behaviour I prefer is something more like the old customizer:
 https://getbootstrap.com/docs/3.3/customize/#download

**Comment by Johann-S** [comment]
Ok I understand, more clicks 😟 

Feel free to push a PR about that 👍

**Comment by XhmikosR** [comment]
Yeah, I agree. This should be simpler for sure. We could inject any info below the buttons if needed.
